frontend/src/app/choreography/choreography.component.ts

```diff
@@ -232,4 +216,38 @@ export class ChoreographyComponent implements OnInit {
   removePersonFromCarpet(index: number): void {
     this.clearItem(this.choreography.frames[this.activeFrame].subframes[this.activeSubframe].grid[index]);
   }
+
+  clearItem(item: ChoreographyItem): void {
+    item.text = null;
```

Comment:
**suggestion:** extract this logic somewhere to be able to reset a `ChoreographyItem` instance anywhere